apidoc/Titanium/UI/View.yml

```
@@ -1332,8 +1332,11 @@ properties:
   - name: borderRadius
     summary: Radius for the rounded corners of the view's border.
     description: Each corner is rounded using an arc of a circle.
-    type: Number
+        Values for each corner can be specified. For example, '20px 20px' will set both left and right corners to `20px`.
+        Specifying '20px 20px 20px 20px' will set top-left, top-right, bottom-right and bottom-left corners in that order.
+    type: [Number, String, Dictionary<Number>, Dictionary<String>]
```

I think this is meant to be:
```suggestion
    type: [Number, String, Array<Number>, Array<String>]
```